client/app/scripts/liveblog-bloglist/views/main.html

```diff
@@ -90,7 +89,8 @@ <h3 translate>Create New Blog</h3>
             <div class="upload-progress" ng-show="progress.width">
                 <div class="bar" style="width: {{ progress.width }}%;"></div>
             </div>
-            <div class="modal-body" style="height: 470px; max-height:470px;">
+            <div class="modal-body" style="min-height: 300px;">
```

Comment:
For the next time you see one,
static style in html must not happen. :)
I know that you removed one of them, but, while we're at it, we should remove all of them.